sopel/modules/instagram.py

```diff
@@ -39,11 +39,20 @@ def instaparse(bot, trigger, match):
     try:
         bot.say(parse_insta_json(json))
     except ParseError:
-        LOGGER.exception(
-            "Unable to find Instagram's payload for URL %s", instagram_url)
-        bot.say(
-            "Unable to parse this Instagram URL. "
-            "It's probably a temporary error; try again in a bit.")
+        try:
```

Comment:
Also log info is always helpful. Maybe `debug` instead of `info`, idk.

```suggestion
        try:
            LOGGER.info(
                "Falling back to OEmbed metadata; "
                "Sopel's IP may be blocked by Instagram.")
```

Reply:
I'm moving this log output somewhere else.